book/content/software_checklist/01/publishing_results.md

```diff
@@ -0,0 +1,60 @@
+# Publishing Scientific Results
+
+## Ready-to-go demos
+For many projects, we will prepare attractive demos. We want to be able to show a working demo at any moment in time. Therefore, we want to have special branches in git that contain fully stand-alone demos, including a slide deck, that can just be checked out and used directly.
+
```

Comment:
This looks to me like eScience Center specific. It could be generic, but would need to be rephrased.

Reply:
section was removed